docs/sources/reference/components/prometheus/prometheus.exporter.cloudwatch.md

```diff
@@ -279,7 +279,7 @@ on how to explore metrics, to easily pick the ones you need.
 controls how far back in time CloudWatch metrics are considered during each {{< param "PRODUCT_NAME" >}} scrape.
 If both settings are configured, the time parameters when calling CloudWatch APIs works as follows:
 
-{{< figure src="/media/docs/alloy/cloudwatch-period-and-length-time-model-2.png" >}}
+{{< figure src="/media/docs/alloy/cloudwatch-period-and-length-time-model-2.png" alt="An example of a Cloudwatch period and length time model" >}}
```

Comment:
It's CloudWatch and we should use Amazon CloudWatch in the first instance: https://grafana.com/docs/writers-toolkit/write/style-guide/capitalization-punctuation/#amazon-products

Reply:
Oh.. I missed that. I'll get a PR up for that.

Reply:
No worries, drive by nit :)